src/overlays/actors/ovl_kaleido_scope/z_kaleido_item.c

```diff
@@ -49,7 +50,7 @@ void KaleidoScope_DrawAmmoCount(PauseContext* pauseCtx, GraphicsContext* gfxCtx,
     if (i != 0) {
         gSPVertex(POLY_OPA_DISP++, &pauseCtx->itemVtx[(sAmmoVtxOffset[item] + 27) * 4], 4, 0);
 
-        gDPLoadTextureBlock(POLY_OPA_DISP++, D_020035C0[i], G_IM_FMT_IA, G_IM_SIZ_8b, 8, 8, 0,
+        gDPLoadTextureBlock(POLY_OPA_DISP++, ((u8*)gHUDAmmo0Tex + (8 * 8 * i)), G_IM_FMT_IA, G_IM_SIZ_8b, 8, 8, 0,
```

Comment:
The names for all these HUD symbols used in the pause menu should probably be adjusted

Reply:
How should they change?

Reply:
I think we could just drop the HUD part of it, or use a name that fits both the hud and the pause menu.

Reply:
gAmmoDigit0Tex ?